Makefile

```diff
@@ -734,7 +734,11 @@ OPTIMIZE    = -O0
 LTO_FLAGS   = $(OPTIMIZE)
 else
 OPTIMIZE    = -Os
+ifeq ($(TARGET),$(filter $(TARGET),$(F1_TARGETS)))
```

Comment:
It might be added in the future when the real bug would be solved to why this optimiser messes up the DMA buffers.